src/components/TeamRegistrationForm.tsx

```diff
@@ -44,30 +45,25 @@ function getFileAsBase64(file) {
   });
 }
 
-export default function TeamRegistrationForm({ handles, wardens, className }) {
+export default function TeamRegistrationForm({
+  handles,
+  wardens,
+}: TeamRegistrationFormProps) {
   const { currentUser } = useUser();
   const { user, isInitialized } = useMoralis();
 
   const [state, setState] = useState(initialState);
-  const [hasValidationErrors, setHasValidationErrors] = useState(false);
-  const [status, setStatus] = useState<FormStatus>(FormStatus.Unsubmitted);
-  const [errorMessage, setErrorMessage] = useState<string | ReactNode>("");
   const [teamMembers, setTeamMembers] = useState<
-    { value: string; image: string }[]
+    { value: string; image: unknown }[]
```

Comment:
Something like this, for the string field that might not exist?

```suggestion
    { value: string; image: string | undefined }[]
```

Reply:
The reason I changed it to `unknown` instead of `string` is actually because it's not a string. It's an object and the image src itself is `image.childImageSharp.resize.src`. I didn't feel like defining all of that, so I just made it `unknown`.

Reply:
Oops, it is actually an image then -- ok! :)